app/parse.py

```diff
@@ -8,8 +14,66 @@ class Quote:
     tags: list[str]
 
 
-def main(output_csv_path: str) -> None:
-    pass
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select(".tag")],
+    )
+
+
+def get_single_page_quote(page_soup: BeautifulSoup) -> [Quote]:
```

Comment:
The return type annotation `[Quote]` is incorrect. It should be `list[Quote]` to indicate that the function returns a list of `Quote` objects.